subscriptions/tasks.py

```diff
@@ -12,11 +12,40 @@
 from .models import Subscription
 from contentstore.models import Message
 from scheduler.client import SchedulerApiClient
+from go_http.metrics import MetricsApiClient
 
 logger = get_task_logger(__name__)
 
 
-class Send_Next_Message(Task):
+def get_metric_client(self, session=None):
+    return MetricsApiClient(
+        auth_token=settings.METRICS_AUTH_TOKEN,
+        api_url=settings.METRICS_URL,
+        session=session)
+
+
+class FireMetric(Task):
+
+    """ Fires a metric using the MetricsApiClient
+    """
+
+    def run(self, metric_name, metric_value, session=None, **kwargs):
+        metric_value = float(metric_value)
+        metric = {
+            metric_name: metric_value
+        }
+        try:
```

Comment:
Can we take this generic `try` out so we get an exception in Sentry. If we see specific types we want to handle we'll add them in later